src/store/logviewer/sagas.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const placeholder=true
```

Comment:
Consider moving websocket connection handling to sagas here

Reply:
if I move it here, will it run immediately when the react app spins up? Cause that's what I'd like.

Reply:
Yes, you can dispatch an action to kick start the saga as soon as the app starts, regardless of tabs. See `src/index.tsx` for an example where we kick off fetching config.

Reply:
Same w/ getting saga set up, will do in seperate pr